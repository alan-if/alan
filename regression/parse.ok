Input parse test: 
    v - single verb 
    vo - verb & object 
    voo - verb & object 
    vowo - verb & object 'with' object 

Also try 'them', 'it', 'all' etc.


L.
There is a o1, a o2, a o3 and a o4 here.

> Verb v.

> Verb vo on o1.

> Verb vo on o2.

> Verb vo on o2.

> I don't know what you mean by 'them'.

> (o1) Verb vo on o1.

(o2) Verb vo on o2.

(o3) Verb vo on o3.

(o4) Verb vo on o4.

> (o1) Verb vo on o1.

(o2) Verb vo on o2.

(o3) Verb vo on o3.

(o4) Verb vo on o4.

> Verb voo on o1 and o2.

> I don't know what you mean by 'them'.

> (o1) Verb voo on o1 and o3.

(o2) Verb voo on o2 and o3.

(o3) Verb voo on o3 and o3.

(o4) Verb voo on o4 and o3.

> (o1) Verb voo on o1 and o3.

(o2) Verb voo on o2 and o3.

(o3) Verb voo on o3 and o3.

(o4) Verb voo on o4 and o3.

> (o1) Verb voo on o1 and o3.

(o2) Verb voo on o2 and o3.

(o3) Verb voo on o3 and o3.

(o4) Verb voo on o4 and o3.

> Verb vowo on o1 with o2.

> I don't know what you mean by 'them'.

> (o1) Verb vowo on o1 with o2.

(o2) Verb vowo on o2 with o2.

(o3) Verb vowo on o3 with o2.

(o4) Verb vowo on o4 with o2.

> You can't do that.

> 

Do you want to restart (y) ? 
