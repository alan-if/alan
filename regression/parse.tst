../compiler/alan parse
../interpreter/arun -n parse < parse.input
