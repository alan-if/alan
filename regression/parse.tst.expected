########## parse ##########
alan parse
arun parse < parse.input
########## /bin/sh  parse.tst ##########

        No warnings or errors detected.
        1 informational message(s).

Input parse test: 
    v - single verb 
    vo - verb & object 
    voo - verb & object 
    vowo - verb & object 'with' object 

Also try 'them', 'it', 'all' etc.


L.
There is a o1, a o2, a o3 and a o4 here.

> v
Verb v.

> v o1
Verb vo on o1.

> v o2
Verb vo on o2.

> v it
Verb vo on o2.

> v them
I don't know what you mean by 'them'.

> v all
(o1) Verb vo on o1.

(o2) Verb vo on o2.

(o3) Verb vo on o3.

(o4) Verb vo on o4.

> v them
(o1) Verb vo on o1.

(o2) Verb vo on o2.

(o3) Verb vo on o3.

(o4) Verb vo on o4.

> v o1 o2
Verb voo on o1 and o2.

> v them o3
I don't know what you mean by 'them'.

> v all o3
(o1) Verb voo on o1 and o3.

(o2) Verb voo on o2 and o3.

(o3) Verb voo on o3 and o3.

(o4) Verb voo on o4 and o3.

> v them o3
(o1) Verb voo on o1 and o3.

(o2) Verb voo on o2 and o3.

(o3) Verb voo on o3 and o3.

(o4) Verb voo on o4 and o3.

> v them it
(o1) Verb voo on o1 and o3.

(o2) Verb voo on o2 and o3.

(o3) Verb voo on o3 and o3.

(o4) Verb voo on o4 and o3.

> v o1 with o2
Verb vowo on o1 with o2.

> v them with it
I don't know what you mean by 'them'.

> v all with o2
(o1) Verb vowo on o1 with o2.

(o2) Verb vowo on o2 with o2.

(o3) Verb vowo on o3 with o2.

(o4) Verb vowo on o4 with o2.

> v 01 and 02 and 03 with 04
You can't do that.

> 

Do you want to RESTART, RESTORE or QUIT ? 
